include/settings.h

```diff
@@ -1,4 +1,4 @@
-#define GOLDCHEATS_VERSION          "1.0.0"     //GoldCheats PS4 version (about menu)
+#define GOLDCHEATS_VERSION          "1.1.0"     //GoldCheats PS4 version (about menu)
```

Comment:
```suggestion
#define GOLDCHEATS_VERSION          "1.0.2"     //GoldCheats PS4 version (about menu)
```

there are no major changes so honestly I'd wait for more features before a release, but if we still want to release then I'd just use a minor version like 1.0.2 or 1.0.4

Reply:
That is fair 👍